proposals/0050-increase-prioritization-fee.md

```diff
@@ -0,0 +1,103 @@
+---
+simd: 0050
+title: Increase Prioritization Fee
+authors:
+  - Tao Zhu
+category: Standard
+type: Core
+status: Draft
+created: 2023-05-22
+feature: #31453
+---
+
+## Summary
+
+Cost of prioritizing should be a meaningful portion of transaction base fee.
+
+## Motivation
+
+Prioritization fee is intended to be used sensitively during congestion,
+provides advanced users a tool to access particular resources when contented
+with a meaningful fee, while other users may opt to delay access to avoid
+paying extra fee. When used properly, it helps to reduce network congestion.
```

Comment:
made an update, highlights "while other users may opt to delay access to avoid paying extra fee" via economic incentives. When account (or block) is contented, with local fee market, some users might decide isn't worth high prio fee to send their txs at that time, therefore helps to reduce congestion from sources.

Reply:
why wouldn't they just not pay the fee, and send an unprioritized transaction?

Reply:
Unprioritized transactions most likely will not have chance to be "tried" by leader, so not being able to add more contention to resources. quic would help on network spamming.

Reply:
> Unprioritized transactions most likely will not have chance to be "tried" by leader

That really depends on how contested the accounts are. I guess the only reason they'd not submit is if they saw the **minimum** fee was > 0 for the account(s) or block.

Reply:
> I guess the only reason they'd not submit is if they saw the minimum fee was > 0 for the account(s) or block.

yea, or similar scenario: https://github.com/solana-foundation/solana-improvement-documents/pull/50#issuecomment-1589485621

One fictional trading setup wrt priority fee could be: 
-  have one separate process keep pulling min prio fee for block and relevant account(s) from RPC endpoint, build local historical "fee market"s. 
- develop algos to utilize local historical data to predict prio fee for outgoing transactions. Perhaps something like "for Account `x`, with urgency of `y`, under market condition `z`, what should prioritization fee be?" 
- the trading process uses predicted prio fee for outbound transactions.

This requires the integrity of local fee market, which is part of the aiming of this proposal.